lib/graphql/schema/validation.rb

```diff
@@ -201,6 +214,17 @@ def self.assert_named_items_are_valid(item_name, get_items_proc)
           Rules::SCHEMA_CAN_GENERATE_IDS,
         ],
       }
+
+      INTROSPECTION_TYPES = Hash[[
```

Comment:
I had initially tried building this list dynamically with `Schema::ReduceTypes` but it was already _too late_ since validation kicks in at that point.

On the flip side, any new type we add to the introspection schema will necessarily have to show up here otherwise tests will fail so we won't forget.

Is there a more elegant solution?